### PR TITLE
Create component for n/a label

### PR DESCRIPTION
**Pull request by marians**
This PR introduces a component for the `n/a` label that so far was created in various ways, usually as a simple string.

The appearance is tweaked to look "dimmed", to be distinguishable from labels showing actual information.

There is a storybook story under `Display` / `NotAvailable`.

### CODE REVIEW

File: src/components/UI/Display/NotAvailable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    <NotAvailableOuter {...rest}>
+      <abbr aria-label='no information available'>n/a</abbr>
+    </NotAvailableOuter>
+  );
+};
+
+NotAvailable.propTypes = {};
+
+export default NotAvailable;
```
Comment (marians) [inline src/components/UI/Display/NotAvailable/index.tsx:25]: Here is the component defiition.

File: src/components/Cluster/ClusterDetail/ClusterApps.js [hunk 2]
```diff
@@ -240,19 +245,19 @@ class ClusterApps extends React.Component {
...
+      version: null,
     },
     {
       name: 'RBAC and PSP defaults',
       logoUrl: '/images/app_icons/rbac_and_psp_defaults@2x.png',
       category: 'essentials',
-      version: 'n/a',
+      version: null,
     },
   ];
```
Comment (marians) [inline src/components/Cluster/ClusterDetail/ClusterApps.js:262]: I'm not so confident about this change. Does it look sane? Where can I test this manually?

Comment (marians) [reply src/components/Cluster/ClusterDetail/ClusterApps.js:262]: Testing on gollum. Currently there is this:

image

Should there be an n/a label appearing?

Comment (marians) [reply src/components/Cluster/ClusterDetail/ClusterApps.js:262]: Fixed it.

image

File: src/components/UI/Display/NotAvailable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+const NotAvailableOuter = styled.span`
+  opacity: 0.5;
+`;
+
+interface INotAvailableProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+const NotAvailable: FC<INotAvailableProps> = ({ ...rest }) => {
+  return (
+    <NotAvailableOuter {...rest}>
```
Comment (axbarsan) [inline src/components/UI/Display/NotAvailable/index.tsx:17]: No need to use the rest operator twice

```suggestion
const NotAvailable: FC<INotAvailableProps> = (props) => {
  return (
    <NotAvailableOuter {...props}>
```

**Review by axbarsan — APPROVED** [review]
Me gusta

**Pull request merged**